### ISSUE TITLE
Migrate to dokku1

### ISSUE THREAD

**Issue by ghickman** [ebmdatalab/trialstracker#14]
We want to [move some sites](https://github.com/opensafely-core/security/issues/2) from dokku2 to dokku1.

Done:
* Update references to dokk2 to dokku1

ToDo:
* [x] Move dokku2 deploy into maintenance mode
* [x] Copy production db over
* [x] Flip DNS
* [x] Remove deploy on dokku2

**Comment by ghickman** [comment]
This site uses nginx as the base image which exposes on port 80, we had to map that instead of the default port 5000 (discovered with `dokku ps` h/t @bloodearnest).  We fixed that by running:

```
dokku proxy:ports-clear trials-tracker
dokku proxy:ports-add trials-tracker http:80:80 https:443:80
```

That got the site up and running but we're going to move that configuration into the Dockerfile using `EXPOSE`.

**Comment by ghickman** [comment]
This site now has a LetsEncrypt provisioned certificate via Dokku, and the correct ports added via `EXPOSE`.  We had to remove all the ports, and drop the env var (`dokku proxy:ports` is a wrapper for setting that), then redeploy and provision a new certificate to get everything set up automatically.  We opted for that route just in case doing the manual step above was different (it wasn't but best to have checked).
